src/js/themes/base.js

```diff
@@ -1501,6 +1508,18 @@ export const generate = (baseSpacing = 24, scale = 6) => {
       xlarge: { ...fontSizing(2) },
       xxlarge: { ...fontSizing(4) },
     },
+    thumbsRating: {
```

Comment:
I can strip these out just left them incase we want to see story and change naming

Reply:
I'm inclined to not have these colors here as part of the base theme.

Reply:
yes I can remove you okay with theme structure?